pages/common/lorem.md

```diff
@@ -0,0 +1,19 @@
+# lorem
+
+> Create more or less random lorem ipsum text.
+
+- Print the number of words you want:
```

Comment:
Generally, we try to avoid using `you` in tldr-pages. Perhaps we can rephrase this?
```

Reply:
Perhaps:
```suggestion
- Print the specified number of words:
```